roles/installer/tasks/main.yml

```diff
@@ -11,6 +11,10 @@
   when:
     - ldap_cacert_secret != ''
 
+- name: Load Custom Setting Configmap
+  include_tasks: tower_custom_settings.yml
+  when: tower_custom_settings != ''
```

Comment:
We have a pattern that we follow for other secrets where we look for the default name, and allow the user to supply a custom name if they want to. May want to do that here.